bin/load-scxa-analytics.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -e
+
+[ -z ${CONDENSED_SDRF_TSV+x} ] && echo "CONDENSED_SDRF_TSV env var is needed." && exit 1
+
+export SCHEMA_VERSION=6
+export SOLR_COLLECTION=scxa-analytics
+export SOLR_PROCESSORS=${SOLR_COLLECTION}-v${SCHEMA_VERSION}_dedupe,${SOLR_COLLECTION}-v${SCHEMA_VERSION}_ontology_expansion
+
+BASE_FILENAME=`basename $CONDENSED_SDRF_TSV .tsv`
+
+COND_SDRF_JSONL_FILENAME=$BASE_FILENAME.jsonl
+echo "Transform condensed SDRF file to JSONL -> $COND_SDRF_JSONL_FILENAME"
+condSdrf2tsvForSCXAJSONFactorsIndex.sh $CONDENSED_SDRF_TSV > $COND_SDRF_JSONL_FILENAME
```

Comment:
We should have here an env variable to set the working directory where this intermediate file is generated. This will avoid issues between parallel executions and make use of the faster scratch spaces, which are different to the longer term shared fs and the software fs.

Reply:
Applicable to all intermediate files, can be same WD for all of them of course.